scripts/maintenance/computational-clusters/autoscaled_monitor/core.py

```diff
@@ -152,13 +153,18 @@ def _print_dynamic_instances(
             f"{_create_graylog_permalinks(environment, instance.ec2_instance)}",
             end_section=True,
         )
-    rich.print(table, flush=True)
+    if output:
+        with output.open("w") as fp:
+            rich.print(table, flush=True, file=fp)
```

Comment:
Q1: why rich for files?
Q2: pro/cons for `rich.` vs directy `typer.secho`  ? I thought the latter could integrate `rich`

Reply:
I am using rich in this script everywhere. that is why.
Now where did you see that secho was using rich?
![image](https://github.com/user-attachments/assets/dc099204-df07-4c3d-918b-5726869064c1)
It does not seem to be so.

They recommend to use rich for printing stuff. but I guess yeah that does not include my case.

Reply:
well actually I wanted to have it with the same layout in file or not in file